src/index.css

```diff
@@ -0,0 +1,247 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap');
+
+body {
```

Comment:
*I strongly suggest You follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Colors (select one main color for your website).
Typography: font face, size and weight.
Layout: composition and space between elements.
## currently your design doesn't follow the provided one 
![image](https://github.com/ahmed025566/financial-eye/assets/52497246/3bb5ca0c-3102-4532-80d7-41dc661766a6)![image](https://github.com/ahmed025566/financial-eye/assets/52497246/e9447f01-f43e-49c2-ae85-337c524465c8)

## design provided
![image](https://github.com/ahmed025566/financial-eye/assets/52497246/977840fb-e04d-482a-ac9a-7e9fe116b531)